package.json

```diff
@@ -4,18 +4,19 @@
   "description": "Control your grow environment.",
   "main": "index.js",
   "engines": {
-    "node": "12.16.1"
+    "node": "14.16.1"
```

Comment:
The modules failed to install / app failed to build with this node version for me. I bumped this up to the next LTS version. Is that okay?